data/vishal126.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Vishal Menariya",
+  "type": "personal",
+  "bio": "jai shree ram",
+  "links": [
+    {
+      "name": "Follow me on FACEBOOK",
+      "url": "https://www.facebook.com/vishal.menaria.3956/",
+      "icon": "FaFacebook"
+    },
+    {
+      "name": "Follow me on google",
+      "url": "www.google.com",
```

Comment:
I am not sure what you mean by adding the Google URL as this is not a link to one of your social media profiles. 

As this is not a blocker I will merge this Pull Request. However if you would like to make changes to your Profile, please raise a new Pull Request.  I would suggest you take a look at our Editing Guide in Docs: https://linkfree.io/docs/how-to-guides/editing